### ISSUE TITLE
[Feature] Support ONCE Dataset

### ISSUE THREAD

**Issue by OrangeSodahub** [open-mmlab/mmdetection3d#2259]
### What's the feature?

Another issue #1353 had called for this feature. And I drafted one #2257

### Any other context?

Now it looks like does well in train/val/test pipeline.
But I'm confused about the unified coordinate system in mmdet3d. According to lidar_box defined in mmdet3d.core.structer, it's x-front, y-left, z-up, so I first transform the once (which is x-left, y-back, z-up) points and annotated boxes3d to the standard lidar coordinate by applying a self-defined matrix but it occurred bit problem when decreasing loss curve. Then I delete the all transformation (like pcdet does), then the training loss looks better but still cannot get approprate results.
Big question is, why you specify the coord_type of kitti/nuscenes/... dataset to `LiDAR` though they have different coordinate system, and not apply extra transformation on gt_boxes_3d when get training data (like nuscenes). How could it be correct? That's the problem when I develop this feature. Hope your help and thanks!

**Comment by JingweiZhang12** [comment]
@OrangeSodahub Thanks for your support!
1. For the box transformation, you can use `browse_dataset` to verify the correctness of the transformed boxes.
2. We unified the coordinate for the convenience of model inference on different datasets.
Maybe we can add WeChat for the subsequent conversation. My: zjw209624

**Comment by OrangeSodahub** [comment]
The latest training log (pointpillars) [here](https://wandb.ai/gzzyyxy/once-dataset-for-mmdet3d/runs/6vt7ftzz/logs?workspace=user-gzzyyxy), really werid.
